templates/field/field--node--oe-consultation-response-button--oe-consultation--full.html.twig

```diff
@@ -0,0 +1,29 @@
+{#
+/**
+ * @file
+ * Respond button link field in the Consultation content full view mode.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% set label = 'Respond to the questionnaire'|t %}
+{% if items.0.content['#url'].external %}
+  {% set path = items.0.content['#url'].uri %}
+  {% if items.0.content['#url'].uri != items.0.content['#title'] %}
+    {% set label = items.0.content['#title'] %}
+  {% endif %}
+{% else %}
+  {% set path = path(items.0.content['#url'].routeName, items.0.content['#url'].routeParameters) %}
+  {% if items.0.content['#url'].routeName != items.0.content['#title'] %}
+    {% set label = items.0.content['#title'] %}
+  {% endif %}
+{% endif %}
```

Comment:
All these would better in a preprocess that prepares the path variable to pass for the template.